package-lock.json

```diff
@@ -2205,12 +5943,9 @@
       "dev": true
     },
     "prismjs": {
-      "version": "1.23.0",
```

Comment:
Per `Dependabot alerts` in github `security`, https://github.com/awslabs/qldb-workbench/security/dependabot/package-lock.json/prismjs/open, we only need to update `prismjs` to a non-vulnerable version >= 1.24.0.  Do we want to add more packages here?

Reply:
All these changes are resulted automatically by running `npm audit fix`. NPM audit actually found two security vulnerabilities, one of them is not reported by DependentBot yet. I think by updating `prismjs`, the tool automatically upgrades other packages as well.

Reply:
Good to know that. Thanks!